crates/ruff_linter/src/rules/airflow/snapshots/ruff_linter__rules__airflow__tests__AIR302_AIR302_args.py.snap

```diff
@@ -11,46 +11,76 @@ AIR302_args.py:15:39: AIR302 `schedule_interval` is removed in Airflow 3.0; use
 16 | 
 17 | DAG(dag_id="class_timetable", timetable=NullTimetable())
    |
+   = help: Replace deprecated keywords in Airflow 3.0
 
-AIR302_args.py:17:31: AIR302 `timetable` is removed in Airflow 3.0; use `schedule` instead
+ℹ Unsafe fix
+12 12 | 
+13 13 | DAG(dag_id="class_schedule", schedule="@hourly")
+14 14 | 
+15    |-DAG(dag_id="class_schedule_interval", schedule_interval="@hourly")
+   15 |+DAG(dag_id="class_schedule_interval", schedule="@hourly")
+16 16 | 
+17 17 | DAG(dag_id="class_timetable", timetable=NullTimetable())
+18 18 | 
+
+AIR302_args.py:17:31: AIR302 [*] `timetable` is removed in Airflow 3.0; use `schedule` instead
    |
 15 | DAG(dag_id="class_schedule_interval", schedule_interval="@hourly")
 16 | 
 17 | DAG(dag_id="class_timetable", timetable=NullTimetable())
    |                               ^^^^^^^^^ AIR302
    |
+   = help: Replace deprecated keywords in Airflow 3.0
```

Comment:
What do you think of using "use `schedule` instead` as help text and remove it from the message

Reply:
Agreed, it makes it cleaner to show what is the error in the message and `help` acting as a how to fix it.

```
cargo run -p ruff -- check --preview --select=AIR302 example_latest_only.py --no-cache        

example_latest_only.py:30:5: AIR302 `schedule_interval` is removed in Airflow 3.0.
   |
28 | with DAG(
29 |     dag_id="latest_only",
30 |     schedule_interval="daily",
   |     ^^^^^^^^^^^^^^^^^ AIR302
31 |     start_date=datetime.datetime(2021, 1, 1),
32 |     catchup=False,
   |
   = help: Use `schedule` instead.

Found 1 error.
No fixes available (1 hidden fix can be enabled with the `--unsafe-fixes` option).
```